builder/softlayer/step_capture_image.go

```diff
@@ -4,35 +4,81 @@ import (
 	"fmt"
 	"github.com/mitchellh/multistep"
 	"github.com/mitchellh/packer/packer"
+	"strings"
 )
 
 type stepCaptureImage struct{}
 
+func (self *stepCaptureImage) getBlockDeviceIds(blockDevices []interface{}) ([]int64) {
+	blockDeviceIds := make([]int64, len(blockDevices))
+	deviceCount := 0
+
+	for _, val := range blockDevices {
+		blockDevice := val.(map[string]interface{})
+		diskImage := blockDevice["diskImage"].(map[string]interface{})
+		name := diskImage["name"].(string)
+		id := int64(blockDevice["id"].(float64))
+
+		if !strings.Contains(name, "SWAP") {
+			blockDeviceIds[deviceCount] = id
+			deviceCount++
+		}
+	}
+
+	return blockDeviceIds[:deviceCount]
+}
+
```

Comment:
Can we move this function to `client.go`?

It seems that since `getBlockDevices` function is already there, it is more logical to move this function there too.